staging/src/k8s.io/client-go/tools/cache/listwatch.go

```diff
@@ -97,7 +101,11 @@ func NewListWatchFromClientWithMultiTenancy(c Getter, resource string, namespace
 // Option modifier is a function takes a ListOptions and modifies the consumed ListOptions. Provide customized modifier function
 // to apply modification to ListOptions with a field selector, a label selector, or any other desired options.
 func NewFilteredListWatchFromClient(c Getter, resource string, namespace string, optionsModifier func(options *metav1.ListOptions)) *ListWatch {
-	return NewFilteredListWatchFromClientWithMultiTenancy(c, resource, namespace, optionsModifier, metav1.TenantSystem)
+	tenant := metav1.TenantSystem
```

Comment:
Isn't Sonya using default tenant, which is system tenant, in her testing?

Reply:
This change is that when watching/listing cluster-scope resources like "nodes", the url link should be "/api/v1/nodes", instead of "/api/v1/tenants/system/nodes".  Also, in cases that the K8s would like to list/watch across all the namespaces (which means the whole cluster in K8s), we should let it do across all the tenants ( which is what whole cluster means in Arktos).

It is not about which tenant you are using.

Reply:
"/api/v1/tenants/system/nodes" looks a totally legitimate path to me. What's wrong here?